app/ui/src/Components/Icons/ProjectIcon/ProjectIcon.tsx

```diff
@@ -2,14 +2,18 @@ import { SvgIcon, SvgIconProps } from '@material-ui/core';
 import React from 'react';
 import styles from './ProjectIcon.module.scss';
 import cx from 'classnames';
-import { ProjectState } from '../../../Graphql/types/globalTypes';
 
 interface AdditionalProps {
-  state: ProjectState;
+  isArchived: boolean;
 }
 
 const ProjectIcon = (props: AdditionalProps & SvgIconProps) => (
-  <SvgIcon {...props} className={cx(props.className, styles[props.state])}>
+  <SvgIcon
+    className={cx(props.className, {
+      [styles.STARTED]: !props.isArchived,
+      [styles.ARCHIVED]: props.isArchived,
+    })}
```

Comment:
Started is no longer a state, use only archived

```suggestion
    className={cx(props.className, {
      [styles.ARCHIVED]: props.isArchived,
    })}
```